regression/contracts/variant_missing_invariant_warning/test.desc

```diff
@@ -1,10 +1,7 @@
 CORE
 main.c
 --apply-loop-contracts
-activate-multi-line-match
 ^The loop at file main.c line 4 function main does not have a loop invariant, but has a decreases clause. Hence, a default loop invariant \('true'\) is being used.$
-^\[main.1\].*Check loop invariant before entry: SUCCESS$
-^\[main.2\].*Check that loop invariant is preserved: SUCCESS$
```

Comment:
Saswat suggested that I should only keep those regular expressions that are actually relevant to what I want to test (that is, decreases clauses). So I decided to remove those lines that are about loop "invariants" (but not loop variants). Having said that, loop invariants are still (usually) necessary for termination proofs, so it would also make sense to keep those lines. Any suggestion?

Reply:
Yes, my point was mainly about the counts that you had `(X of Y)` assertions etc.

I agree with Felipe that we should keep the assertions related to invariants, since they are going to be checked for variants as well.

Reply:
OK. I just talked with Saswat, and apparently, I removed too many lines from regular expressions in regression tests. I will add them back!